src/test/java/StringTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    @DisplayName("요구사항1 - split() 메서드를 활용해 , 기준으로 분리한 배열을 반환")
+    void stringSplitTest() {
+        String[] value = "1,2".split(",");
+        assertThat(value).containsExactly("1", "2");
+
+        String[] value2 = "1".split(",");
+        assertThat(value2).contains("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).contains("1");
```
이 같이 간단한 구현은 로컬 변수 없이 구현하는 것은 어떨까?